### PR TITLE
Group

### PR DESCRIPTION
**Pull request by carycheng**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @carycheng, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: boxsdk/client/client.py [hunk 1]
```diff
@@ -391,3 +392,33 @@ def get_url(self, endpoint, *args):
         """
         # pylint:disable=no-self-use
         return self._session.get_url(endpoint, *args)
+
+
+    def groups(self, name=None, offset=None, limit=None, fields=None):
```
Comment (jmoldow) [inline boxsdk/client/client.py]: You didn't use `name`.

File: boxsdk/object/group.py [hunk 2]
```diff
@@ -81,3 +82,33 @@ def add_member(self, user, role):
         response = box_response.json()
 
         return GroupMembership(self._session, response['id'], response, user=user, group=self)
+
+
+    def collaborations(self, offset=None, limit=None):
```
Comment (jmoldow) [inline boxsdk/object/group.py]: Add `fields`

File: boxsdk/object/user.py [hunk 3]
```diff
@@ -2,10 +2,42 @@
...
 from .base_object import BaseObject
+from ..pagination.limit_offset_based_object_collection import LimitOffsetBasedObjectCollection
 
 
 class User(BaseObject):
     """Represents a Box user."""
 
     _item_type = 'user'
+
+    def get_memberships(self, offset=None, limit=None):
```
Comment (jmoldow) [inline boxsdk/object/user.py]: Add  `fields`

File: boxsdk/pagination/page.py [hunk 4]
```diff
@@ -38,7 +39,7 @@ def _translator(self):
 
         :rtype:   :class:`Translator`
         """
-        return self._session.translator
+        return Translator()
```
Comment (jmoldow) [inline boxsdk/pagination/page.py]: Revert

**Pull request closed without merging**